src/kokkos_ext/ArborX_DetailsKokkosExtSort.hpp

```diff
@@ -170,13 +170,18 @@ void sortByKey(Kokkos::Experimental::SYCL const &space, Keys &keys,
   if (n == 0)
     return;
 
-  auto zipped_begin =
-      oneapi::dpl::make_zip_iterator(keys.data(), values.data());
   oneapi::dpl::execution::device_policy policy(
       *space.impl_internal_space_instance()->m_queue);
+#if ONEDPL_VERSION_MAJOR > 2022 ||                                             \
+    (ONEDPL_VERSION_MAJOR == 2022 && ONEDPL_VESION_MINOR >= 2)
```

Comment:
Should we just require this? What's the minimum version for Kokkos? Do we care?

Reply:
`Kokkos` doesn't have a minimum version; as of now, we are only using `dpl::sort` anyway.
`oneDPL` is not necessarily recent enough on the testbeds so I wouldn't want to require 2022.2 at this point.